.github/workflows/event-merge-to-queue.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     uses: ./.github/workflows/task-get-latest-tag.yml
     with:
       repo: redis/redis
-      prefix: '7.4'
+      prefix: '8.0'
```

Comment:
Needed for new API/flag, right?

Reply:
Yes. It is needed for the new flag. But this PR did not modified it, that change is part of #5318 which was merged to master branch.